PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, experiments you ran to see how -->
+<!--- your change affects existing areas of the code and their behaviors, etc. -->
```

Comment:
I am wondering if we should include some tips and tricks for testing changes here? In particular, how to do a local pip install to test Python changes, and how to test JavaScript changes / re-generate minified JS.